qa/setup_packages.py

```diff
@@ -273,7 +280,8 @@ class CudaPackage(BasePackage):
     def __init__(self, key, versions, name=None):
         super(CudaPackage, self).__init__(key, versions, name)
         if not isinstance(versions, dict):
-            raise TypeError("versions argument should by dict type [cuda_version : list_of_versions")
+            raise TypeError("versions argument should by dict type [cuda_version :" +
+                            "list_of_versions")
```

Comment:
Missing space and an unnecessary concatenation
```suggestion
            raise TypeError("versions argument should by dict type [cuda_version : "
                            "list_of_versions")
```